src/org/zaproxy/zap/view/widgets/WritableFileChooser.java

```diff
@@ -26,6 +28,19 @@ public WritableFileChooser(File currentDirectory) {
 	public void approveSelection() {
 		File selectedFile = getSelectedFile();
 
+		if (!selectedFile.exists()) {
```

Comment:
The check that the file exists could be saved in a temporary variable to use in the other if statement.